tests/render_fails.rs

```diff
@@ -142,3 +142,18 @@ fn test_error_in_macro_location() {
         "Failed to render 'error-location/error_in_macro.html': error while rendering macro `macros::cause_error`"
     );
 }
+
+#[test]
+fn test_error_is_send_and_sync() {
```

Comment:
sorry, can you move it to a new test module in src/errors.rs so it's co-located?